src/libraries/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -395,28 +396,28 @@ private bool AgeCookies(string domain)
             }
             lock (m_domainTable.SyncRoot)
             {
-                foreach (DictionaryEntry entry in m_domainTable)
+                foreach (object? entry in m_domainTable)
```

Comment:
This should be changed to avoid casting to `DictionaryEntry` twice in the loop body.

Reply:
Fixed.